uniffi_bindgen/src/bindings/python/gen_python/mod.rs

```diff
@@ -77,24 +78,39 @@ impl<'a> PythonWrapper<'a> {
 
     pub fn members(&self) -> Vec<Box<dyn CodeDeclaration + 'a>> {
         let ci = self.ci;
-        vec![]
-            .into_iter()
-            .chain(ci.iter_enum_definitions().into_iter().map(|inner| {
+        vec![
+            Box::new(callback_interface::PythonCallbackInterfaceRuntime::new(ci))
+                as Box<dyn CodeDeclaration>,
+        ]
+        .into_iter()
```

Comment:
mildly surprised by the indentation changes here? Is that cargo fmt? (I guess the fact the vec now has an element, the close `]` gets moved, and everything else follows?)